src/components/LoginPage.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+
+const LoginPage = () => {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    try {
```

Comment:
Error handling can be improved. Instead of logging the error to console, consider showing the error to the user using a modal or a toast notification.